### PR TITLE
JC-2048 Sign Up link is available for logged in user

### PR DESCRIPTION
**Pull request by vps2**
- when logged user try go to the sign-up page, he will be redirected to
  the main page.

### CODE REVIEW

File: jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/UserControllerTest.java [hunk 1]
```diff
@@ -119,7 +119,8 @@ public void testInitBinderDefaultStringEditor() {
     }
 
     @Test
-    public void testRegistrationPage() throws Exception {
+    public void testRegistrationPageWhenUserNotLogged() throws Exception {
```
Comment (ctapobep) [inline jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/UserControllerTest.java]: Let's name it in BDD style: `userMustBeAbleToOpenRegistrationPage_ifHeIsNotLoggedIn`

File: jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/UserControllerTest.java [hunk 2]
```diff
@@ -128,6 +129,14 @@ public void testRegistrationPage() throws Exception {
     }
 
     @Test
+    public void testRegistrationPageWhenUserLogged() throws Exception {
```
Comment (ctapobep) [inline jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/UserControllerTest.java]: `whenOpeningRegistrationPage_userMustBeRedirectedToMainPage_ifHeIsLoggedIn`

**Comment by ctapobep** [comment]
The rest is fine, so please squash your commits after the changes are introduced and I'll merge it.

**Comment by vps2** [comment]
Ready.

**Comment by ctapobep** [comment]
Merged, thanks!

**Pull request closed without merging**